pipeline/release/body.go

```diff
@@ -23,6 +23,12 @@ const bodyTemplate = `{{ .ReleaseNotes }}
 Automated with [GoReleaser](https://github.com/goreleaser)
 Built with {{ .GoVersion }}`
 
+var bodyTemplateT *template.Template
+
+func init() {
+	bodyTemplateT = template.Must(template.New("release").Parse(bodyTemplate))
```

Comment:
maybe we could change the `bodyTemplate` var to `bodyTemplateText` and this one to `bodyTemplate` only... what do you think?